packages/storage-documents/src/website.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ScanType } from './scan-types';
+import { StorageDocument } from './storage-document';
+
+export interface Website extends StorageDocument {
+    name: string;
+    baseUrl: string;
+    priority: number;
+    discoveryPatterns: string[];
+    knownPages: string[];
+    scanners: ScanType[];
+    domainId: string;
+    owners?: string[];
+    orgCvp?: string;
+    usingUhf?: boolean;
```

Comment:
what is Uhf?

Reply:
It is a shared component used by many sites